File: scala/Sandbox.scala
import org.apache.spark.SparkConf
import org.apache.spark.SparkContext
import org.apache.spark.SparkContext._


object Sandbox {
  val conf = new SparkConf().setMaster("local").setAppName("Sandbox")
  val sc = new SparkContext("local", "Sandbox")


  def main(args: Array[String]) = {
    println("spark!")
  }
}

File: python/sandbox.py
from pyspark import SparkConf, SparkContext

conf = SparkConf().setMaster("local").setAppName("sandbox")

sc = SparkContext(conf=conf)

inputRDD = sc.textFile("/var/log/syslog")
infoRDD = inputRDD.filter(lambda line: "info" in line)

print "number of lines"
print infoRDD.count()
print "first 10 lines"
for line in infoRDD.take(10):
  print line
